roles/openshift_grafana/tasks/main.yaml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Create grafana namespace
+  command: oc create namespace grafana
+
+- name: Switch to grafana project
+  command: oc project grafana
+
+- name: Templatize grafana yaml
+  template: src=grafana-ocp.yaml dest=/tmp/grafana-ocp.yaml
+
+- name: load the grafana taml
+  command: oc new-app -f /tmp/grafana-ocp.yaml
```

Comment:
https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/library/oc_process.py with `create: true`. Provide the namespace as well.

Reply:
ok make sense